xbmc/addons/GUIDialogAddonInfo.cpp

```diff
@@ -336,6 +359,31 @@ void CGUIDialogAddonInfo::OnInstall()
   if (i != -1)
   {
     Close();
+    if (HELPERS::ShowYesNoDialogLines(CVariant{m_item->GetAddonInfo()->Name()},
```

Comment:
we should use `ShowYesNoDialogText` instead, because `CGUIDialogBoxBase::SetLine` do some superfluous split magic and later call `CGUIDialogBoxBase::SetText`. IMO we should think about to mark `ShowYesNoDialogLines` as deprecated.